corehq/apps/app_manager/models.py

```diff
@@ -2289,6 +2289,7 @@ class ModuleBase(IndexedSchema, ModuleMediaMixin, NavMenuItemMediaMixin, Comment
     case_type = StringProperty()
     case_list_form = SchemaProperty(CaseListForm)
     module_filter = StringProperty()
+    put_in_root = BooleanProperty(default=False)
```

Comment:
Could you also update here: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/suite_xml/post_process/workflow.py#L29 ?

Reply:
sure, updated